meetings/2022-11-16.md

```diff
@@ -45,45 +57,103 @@
+* eP: I agree. Especially when we think about AuthZ, one case when data is public or protected info. The paper of Ruben Verborgh and team (ref???) using the case of birthday and so on, it is similar to email address, sometimes you want to share full name, first name... sometimes you want to make everything public. For that information AuthZ should happen before discovery. +1 to take advantage of existing discovery mechanisms to discover data. For different audiences you can select different. vcards they have access to instead of trying every seeAlso blindly. We need to work around some use cases from Ruben's paper and see how it can work, have different proposals. In general I agree discovery needs to be taken into, and preceded with AuthZ for protected information. 
+* DB: Last time in the WebID meeting I remember Tim saying he was interested in having the possibility of a minimum profile. Name, pictures, things apps would consume in a daily basis. Reduce amount of hops for this popular information. Could there be a distinction of priorities in data? 
+* SC: That's an old design pattern that has been in Solid. Just to make it useful. We're talking about Social linked data, the basics, even if it's a pseudonym, it puts something on a screen. If people want to put their real photo in a different location, that's fine. There's a distinction between making some info available for the utility of it, not so much exposing it. You can have anonymous and pseudonymous WebIDs. 
+* WT: I agree that it is important that apps can access important information quickly. But I'd like to raise: 1 - it is quite naive to think that we know what apps will need now or in the future. Even if we decide it's a good idea to have these things, I think they should be in there using some mechanism that relies on the standard discoverable place where they are. You can imagine we store all data even names and avatars in the interop registries, but we let the authz agent/server of interop make it so that if you declare that a certain type of data is public, then ??? is added to your WebID profile. Just brainstorming. 
+* HS: I think we should also be careful about premature optimization in this case. The argument that you following links might be problematic because it might lead you nowhere, that's something you can deal with but expensive. You can have a version 2 of Solid that allows you to specify filters, it could be really complicated, because we're working on consensus... ??? we def could do that, but it's going to take a long time and resources to build. If we look at Mastodon, it's got public profiles. You can put your picture up... so we def wanna get to the point of military grade security but let's avoid tying ourselves into knots. Let's not build again security, that's it. 
```

Comment:
```suggestion
* HS: I think we should also be careful about premature optimization in this case. The argument that following links might be problematic because it might lead you nowhere, that's something you can deal with but expensive. You can have a version 2 of Solid that allows you to specify filters, it could be really complicated, because we're working on consensus... ??? we def could do that, but it's going to take a long time and resources to build. If we look at Mastodon, it's got public profiles. You can put your picture up... so we def wanna get to the point of military-grade security, but let's avoid tying ourselves into knots. Let's not build security again, that's it. 
```